### ISSUE TITLE
Option to skip built-in types

### ISSUE THREAD

**Issue by ViggoV** [2fd/graphdoc#25]
Is it possible to skip the built-in types like scalars and all the underscored stuff like __DirectiveLocation?

**Comment by pelted** [comment]
Was an answer to this ever found?

**Comment by ViggoV** [comment]
I don't believe I ever found a solution, no.. We ended up writing a visual, introspective data browser for our customers instead, which I actually just published the first public draft of: https://github.com/BiQ/graphql-inquirer

**Comment by gmullerb** [comment]
You can use the following plugin [graphdoc-plugin-erase](https://graphdoc-plugins.github.io/docs/graphdoc-plugin-erase.html) to easily achieved a configurable output.
